src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java

```diff
@@ -63,7 +63,8 @@ public boolean canEnchantItem(ItemStack item) {
             if (id.isPresent()) {
                 return (id.get().equals(Items.ADVANCED_CIRCUIT.getItemId()))
                     || (id.get().equals(Items.NANO_BLADE.getItemId()))
-                    || (id.get().equals(Items.GLASS_CUTTER.getItemId()));
+                    || (id.get().equals(Items.GLASS_CUTTER.getItemId()))
+                    || (id.get().equals(Items.LAPOTRON_CRYSTAL.getItemId()));
```

Comment:
Make a god damn Set instead of hundred ORs :pepeCry:

Reply:
idk how